src/collections/blog/2022/2022-06-13-aaditya-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,54 @@
+
+<img src={aadityaFirstPr} alt="Aaditya's first PR merged" />
+
+### A Journey Never to forget 
+
+With this small issue, my journey on Layer5 started. I gradually started learning about the project and started making some impactful contributions. I started to engage in the community more prominently. With the help of these contributions, I learned new technologies, and techniques. Contributing to the codebase gave me so much exposure to React.js, that it would not have been possible if I did it on my own. I took up issues that used React.js and started solving them, and in no time I was comfortable with it. I am thankful, that I didn’t wait for me to learn more about react and then start contributing. I have always applied this principle, learning more by doing actual stuff rather than sitting there and getting stuck in the tutorial loop. 
+
+Another new technology I learned was Jekyll, which is a static site generator. This too was a technology that I had not heard about. I started contributing to issues involving Jekyll and slowly understood the basics and how the laguage works. I was very happy that I was able to make the [compatibility matrix](https://docs.meshery.io/project/compatibility-matrix), using Jekyll. Now even my project for GSoC is written in Jekyll. So, from not knowing what Jekyll is, to making some big changes, did make me happy. 
+
+The community also helped me increase my confidence. The weekly meetings are responsible for this. The way the meeting required me to explain my fixes and give updates regarding the issues I was working on, helped me in getting vocal and comfortable with talking to various people. 
+
+<img src={aadityaPresenting} alt="Aaditya Presenting" /> 
+
+I feel excited that I have been accepted as a Google Summer of Code Intern for this Summer of 2022. I am looking forward to contributing more and making an impact here. The project assigned to me is to make a dashboard at <a href="smp-spec.io/dashboard">SMP Website</a>. Here the results from the CNCF Cluster labs that run [here](https://github.com/layer5io/meshery-smp-action/actions) are to be shown graphically.  I also hope to learn more about Cloud Native technologies and make contributions on the backend too. I also want to state that this will not be the end. I will be engaged with this community as long as I can and will try to give back as much as I can. 
```

Comment:
```suggestion
I am excited to be accepted as a Google Summer of Code Intern for this Summer of 2022. I am looking forward to contributing more and making an impact here. The project assigned to me is to create a dashboard on <a href="smp-spec.io/dashboard">SMP Website</a>. Results from the [CNCF Cluster labs](https://github.com/layer5io/meshery-smp-action/actions) will be displayed here graphically.  I wish to learn more about cloud native technologies and make contributions on the backend too. I also want to state that this will not be the end. I will be engaged with this community as long as I can and will try to give back as much as I can. 
```